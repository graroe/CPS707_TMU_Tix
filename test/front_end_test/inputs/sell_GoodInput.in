login
sales
sell
CirqueDuSoleil
6
sell
CirqueDuSoleil
1
logout