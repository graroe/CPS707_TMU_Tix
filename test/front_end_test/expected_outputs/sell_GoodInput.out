Enter session type <sales or admin>: 
Sales agent login successful.
Enter event name: 
Current number of tickets: 10\nEnter number of tickets to sell: 
6 tickets sold. New total: 4
Enter event name: 
Current number of tickets: 4\nEnter number of tickets to sell: 
1 tickets sold. New total: 3
Session terminated.